src/google/Timeline.js

```diff
@@ -0,0 +1,129 @@
+        this._tag = "div";
+
+        this._data_google = [];
+        this.columns([]);
+        this.data([]);
+    };
+    Timeline.prototype = Object.create(HTMLWidget.prototype);
+
+    /**
+     * Publish Params Common To Other Libraries
+     */
+
+    
+    /**
+     * Publish Params Unique To This Widget
+     */   
+    Timeline.prototype.publish("avoidOverlappingGridLines", true, "boolean", "Whether display elements (e.g., the bars in a timeline) should obscure grid lines. If false, grid lines may be covered completely by display elements. If true, display elements may be altered to keep grid lines visible.",null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("backgroundColor", null, "html-color", "Horizontal axis text style (Font Size)",null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("colors", null, "html-color", "The colors to use for the chart elements. An array of strings, where each element is an HTML color string, for example: colors:['red','#004411'].",null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("enableInteractivity", true, "boolean", "Whether the chart throws user-based events or reacts to user interaction. If false, the chart will not throw 'select' or other interaction-based events (but will throw ready or error events), and will not display hovertext or otherwise change depending on user input.",null,{tags:['Advanced']});
+    Timeline.prototype.publish("forceIFrame", false, "boolean", "Draws the chart inside an inline frame. (Note that on IE8, this option is ignored; all IE8 charts are drawn in i-frames.)",null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("height", null, "number", "Height of the chart, in pixels.",null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("timeline.barLabelStyle", null, "object", "An object that specifies the bar label text style. It has this format: {color: <string>, fontName: <string>, fontSize: <string>}  The color can be any HTML color string, for example 'red' or '#00cc00'",null,{tags:['Basic']});
+
+    Timeline.prototype.publish("timeline.colorByRowLabel", false, "boolean", "If set to true, colors every bar on the row the same. The default is to use one color per bar label.",null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("timeline.groupByRowLabel", true, "boolean", "If set to false, creates one row for every dataTable entry. The default is to collect bars with the same row label into one row.",null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("timeline.rowLabelStyle", null, "object", "An object that specifies the row label text style. It has this format: {color: <string>, fontName: <string>, fontSize: <string>}  The color can be any HTML color string, for example 'red' or '#00cc00' ", null,{tags:['Basic']});
+    
+    Timeline.prototype.publish("timeline.showBarLabels", true, "boolean", "If set to false, omits bar labels. The default is to show them.", null,{tags:['Basic']});
```

Comment:
Yeah, I'm not sure what's up with these parameters that have 'timeline.'. They were on the google parameters table so I copied them.  They do not work as is in out discover params though which is why they are commented out.  Taking the "timeline." off doesn't help, they still don't work.